src/db/h2/lrsql/h2/sql/ddl.sql

```diff
@@ -117,7 +117,7 @@ CREATE TABLE IF NOT EXISTS agent_profile_document (
   last_modified  TIMESTAMP NOT NULL,
   content_type   VARCHAR(255) NOT NULL,
   content_length INTEGER NOT NULL,
-  contents       BINARY NOT NULL,
+  contents       BINARY VARYING NOT NULL,
```

Comment:
i read up the difference. is this just to account for not declaring size?

Reply:
Yes - `BINARY` only works for fixed sized arrays.